cassiopeia/solver/graph_utilities.py

```diff
@@ -166,15 +170,15 @@ def max_cut_improve_cut(G: nx.Graph, cut: List[int]) -> List[int]:
 
 
 def construct_similarity_graph(
-    cm: Dict,
+    character_matrix: np.array,
```

Comment:
same comment about `character_matrix` with string names that gets converted within the function.